app/realtime.php

```diff
@@ -91,13 +91,34 @@
 
 function getDatabase(Registry &$register, string $namespace)
 {
-    $db = $register->get('dbPool')->get();
-    $redis = $register->get('redisPool')->get();
+    $attempts = 0;
+    $sleep = 2;
+    $maxAttempts = 10;
```

Comment:
We might consider having global constants for this to help us make a consistent retry logic across containers.